src/utils/data-mapper.ts

```diff
@@ -209,7 +141,40 @@ const tmdbCollectionMap = {
 const tmdbMovieMap = {
   'belongs_to_collection.id': 'collectionTmdbID',
   'budget': 'budget',
-  'credits': 'credits',
+  'credits': [
+    { key: 'credits' },
+    {
+      key: 'actors',
+      transform: (credits: CreditsResponse | EpisodeCreditsResponse): string[] => {
+        // populate the old "actors" array which came from OpenSubtitles
+        if (!credits?.cast) {
+          return [];
+        }
+        const actors = [];
```

Comment:
here you will have a lot of secondary actors (sometimes more than 100 actors) that will fill the ums db on 'by actor'.

On my java implementation, I use this code to get only the first 5 major actors.
```
	private static ApiStringArray getActors(List<? extends PersonRoleCreditSchema> cast) {
		//search for actors
		Collections.sort(cast, (role1, role2) -> role1.getOrder().intValue() - role2.getOrder().intValue());
		ApiStringArray result = new ApiStringArray();
		for (PersonRoleCreditSchema person : cast) {
			result.add(person.getName());
			if (result.size() > 4) {
				break;
			}
		}
		return result;
	}
```